classes/music_album.rb

```diff
@@ -0,0 +1,17 @@
+require_relative './item'
+
+class MusicAlbum < Item
+  attr_accessor :on_spotify, :archived
+  attr_reader :publish_date
+
+  def initialize(publish_date, on_spotify)
+    super(publish_date)
+    @on_spotify = on_spotify
+  end
+
+  private
+
+  def can_be_archived?
+    super && @on_spotify
+  end
```

Comment:
- [ ] A private method overrides a private method in another file. I don't think this is possible in OOP languages. If I'm correct kindly check the comment above and rectify the issue else I'll like to know if I'm missing something interesting 😄